<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrossDlg</class>
 <widget class="QWidget" name="CrossDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CrossDlg</string>
  </property>
  <widget class="MyLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>240</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>256</width>
     <height>320</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>320</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>256</width>
     <height>320</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 85, 0);
border-width: 2px;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>240</width>
     <height>256</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>256</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>256</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cross">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
